wiki/Registration/en.md

```diff
@@ -1,43 +1,39 @@
+ 5. Continue button                                                                                                                                                    
+3. Log in to osu! and the process is finished. You have successfully registered an osu! account!
 
-<span style="font-size:18px">3. Log in to osu! and the process is finished. You have successfully registered an osu! account! </span>
 
-<img src="Register-3.png" title="fig:Register-3.png" alt="Register-3.png" width="200" /><img src="Login.png" title="fig:Login.png" alt="Login.png" width="120" />
 
-<img src="Userpanel.png" title="Userpanel.png" alt="Userpanel.png" width="320" />
+| Name            | Image                           | Description                                                                                                                                                                                                                                                                                              |
+|:---------------:|:-------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| osu! executable | ![Register.png](Register-3.png) | This executable (which should be named osu!.exe) will open the game itself. If updates are required, they will be applied automatically. _Note: You do not need to open osu! as an administrator. If the game requires it, it will prompt for it itself._                                                |
+| Options panel   | ![Login.png](Login.png)         | If this is your first time starting osu!, you will see this on the left side of your game's screen. Enter your username and password to their respective fields and log in.                                                                                                                              |
+| User panel      | ![Userpanel.png](Userpanel.png) | Once you've entered your username and password correctly, you will see your user panel at the top left of your screen. You are now logged into osu! and ready to play! You will also see your user panel on the bottom of your song selection screen, to the right of your ``Selection Options`` button. |
 
-| Legend                 |
-|------------------------|
-| **1. osu! executable** |
-| **2. Options panel**   |
-| **3. User panel**      |
 
-</center>
 What's next?
 ------------
 
-You're ready to start your rhythm adventure, so proceed to the [welcome](welcome "wikilink") page and start playing! Don't forget to read the [rules](rules "wikilink") very carefully!
+You're ready to start your rhythm adventure, so proceed to the [welcome](/wiki/Welcome/) page and start playing! Don't forget to read the [rules](/wiki/Rules/) very carefully!
```

Comment:
ASG says, "no contractions" (this is to keep the writing a bit more formal);
- Change `You're` to `You are`
- Change `Don't` to `Do not`